Js/index.js

```diff
@@ -4,24 +4,175 @@ let btnCancelMobileMenu = document.querySelector("#btn-close-nav");
+  },
+  {
+    id: 4,
+    url: "./asset/cat1.jpg",
+    title: "Multi-Post Stories Gain+Glory",
+    description: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+  industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type
+  and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and
+  typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+  when an unknown printer took a galley of type and scrambled it 1960s with the relea`,
+    tecnologies: {
+      tech1: "Rubi on rails",
+      tech2: "css",
+      tech3: "JavaScript",
+      tech4: "html",
+    },
+    linkVersion: "https://www.google.com",
+    linkSource: "https://github.com/rebel216/",
+  },
+];
+const WorkDetailsMobile = (index) => `<div class="pop-up">
+  <article class="card-work-detail">
+          <div>
+          <button type="button" class="btn-close-detail">
+            <img src="./asset/Icon-Cancel.svg" alt="cancel icon">
+          </button>
+          </div>
+          <div>
+          <div class="card-image-details">
+              <img src=${dataCards[index].url} alt="cat cute">
+          </div>
+          </div>
+          <div class="card-title-mobile">
+          <h2>
+              ${dataCards[index].title}
+          </h2>
+          </div>
+            <div class="card-links-mobile ">
+                  <a href=${dataCards[index].linkVersion} class="btn-card-details-mobile">
+                  See Live
+                  <img src="./asset/see_live_icon.svg" alt="cat cute">
+                  </a>
+                  <a href=${dataCards[index].linkSource} class="btn-card-details-mobile">
+                  See Sources
+                  <img src="./asset/github_white.svg" alt="cat cute">
+                  </a>
+          </div>
+          <div>
+          <ul class="work-tag-popup">
+              <li class="mobile-tag">${dataCards[index].tecnologies.tech1}</li>
+              <li class="mobile-tag">${dataCards[index].tecnologies.tech3}</li>
+              <li class="mobile-tag">${dataCards[index].tecnologies.tech4}</li>
+          </ul>
+          </div>
+          <div class="popup-paragraph">
+          <p>
+            ${dataCards[index].description}
+          </p>
+          </div>
+        <script src="index.js"></script>
+  </article>
+  </div>
+  `;
+
+for (let i = 0; i < allBtnPopups.length; i++) {
+  allBtnPopups[i].addEventListener("click", () => {
+    const cardWork = document.createElement("div");
+    cardWork.innerHTML = WorkDetailsMobile(i);
+    document.body.appendChild(cardWork);
+    //page.classList.add("noscroll");
+    //alert("cliclked");
+    const btnCloseDetailsMobile = document.querySelector(".btn-close-detail");
+    btnCloseDetailsMobile.addEventListener("click", async () => {
+      document.body.removeChild(cardWork);
+      //page.classList.remove("noscroll");
```

Comment:
- [ ] Please remove all commented code from your codebase to make it clean and maintainable.